File: java/io/stpettersen/blackjack/ISoundEffects.java
/**
 * Blackjack
 * Copyright 2015 Sam Saint-Pettersen
 * Released under the MIT/X11 License.
 * 
 * Java Swing/AWT implementation.
*/

package io.stpettersen.blackjack;

public interface ISoundEffects
{
    boolean toggle();
    void play(String effect);
}

File: scala/src/io/stpettersen/blackjack/ISoundEffects.scala
/**
 * Blackjack
 * Copyright 2015 Sam Saint-Pettersen
 * Released under the MIT/X11 License.
 *
 * Scala Swing/AWT implementation.
 */

package io.stpettersen.blackjack

trait ISoundEffects {
  def toggle: Boolean
  def play(effect: String): Unit
}
